src/main/java/net/atos/zac/util/IndexerenRESTService.java

```diff
@@ -33,36 +25,10 @@ public class IndexerenRESTService {
     @Inject
     private IndexeerService indexeerService;
 
-    @Inject
-    @ActiveSession
-    private Instance<HttpSession> httpSession;
-
     @GET
     @Path("herindexeren/{type}")
-    public HerindexerenInfo herindexeren(@PathParam("type") ZoekObjectType type) {
```

Comment:
If we are getting rid of the `herindexeren` endpoint we also need to update our `reindex-zac-solr-data.sh` script (which I use daily when developing locally to update Solr).